docs/source/conf.py

```diff
@@ -128,6 +135,15 @@
     'analytics_id': 'UA-117752657-2',
 }
 
+
+# html_context = {
+#     "display_github": True,  # Integrate GitHub
```

Comment:
I'm slightly on both sides to remove this. As we have lot of commented out code in `Makefile`  we might as well just keep this.
Or we can totally remove commented out code to keep it clean. let me know!

Reply:
I'd rather revert. We have a lot of commented stuff in `conf.py` because that's how it was initially generated. Config files often come with tons of config options that you can uncomment or tweak to configure the project to your needs. But in this case there's no need to introduce code we don't use.